packit_service/worker/reporting.py

```diff
@@ -25,6 +26,27 @@
 logger = logging.getLogger(__name__)
 
 
+NEWS_FOOTER = [
+    "Wondering what happened during the last year in Packit? "
+    "Take a look at [our blog post](https://packit.dev/posts/2022-features/).",
+    "Do you maintain a Fedora package and don't have access to the upstream repository? "
+    "Packit can help. "
+    "Take a look [here](https://packit.dev/posts/pull-from-upstream/) to know more.",
+    "Do you maintain a Fedora package and you think it's boring? Packit can help. "
+    "Take a look [here](https://packit.dev/posts/downstream-automation/) to know more.",
+    "Want to use a build from a different project when testing? "
+    "Take a look [here](https://packit.dev/posts/testing-farm-triggering/) to know more.",
+]
```

Comment:
Possible future improvements:
* Define this in the packit-service config.
* Load the last n blog posts.
  * This will not allow specifying different text for each post.
  * Also, I am not sure if it's better to promote weekly posts or separate, feature-centric blog-posts.